libraries/AP_OpticalFlow/AP_OpticalFlow_MAV.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "OpticalFlow.h"
+#include <AP_HAL/utility/OwnPtr.h>
+
+class AP_OpticalFlow_MAV : public OpticalFlow_backend
+{
+public:
+    /// constructor
+    AP_OpticalFlow_MAV(OpticalFlow &_frontend) :
```

Comment:
Would be nice to use `using` here

Reply:
OK!  i've switched to using "using" which I've never tried before..